test/cloud-canary/mzcompose.py

```diff
@@ -299,7 +299,11 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 def disable_region(c: Composition) -> None:
     print(f"Shutting down region {REGION} ...")
 
-    c.run("mz", "region", "disable", "--hard")
+    try:
+        c.run("mz", "region", "disable", "--hard")
+    except UIError:
+        # Can return: status 404 Not Found
+        pass
```

Comment:
Are we losing any test coverage by just ignoring the error here? Should we somehow sniff out 404 errors specifically? Should mz perhaps be the one responsible for converting a 404 API error into a successful response?

Reply:
I'd argue `mz` should do so, since that's what it did before https://github.com/MaterializeInc/materialize/pull/30867 landed. @doy-materialize what do you think?

Reply:
That would be my preference as well!